gensim/downloader.py

```diff
@@ -164,6 +164,36 @@ def _calculate_md5_checksum(fname):
     return hash_md5.hexdigest()
 
 
+def _load_info(url=DATA_LIST_URL, encoding='utf-8'):
+    """Load dataset information from the network.
+
+    If the network access fails, fall back to a local cache.  This cache gets
+    updated each time a network request _succeeds_.
+    """
+    cache_path = os.path.join(base_dir, 'information.json')
+
+    try:
+        info_bytes = urlopen(DATA_LIST_URL).read()
+    except (OSError, IOError) as err:
+        #
+        # The exception raised by urlopen differs between Py2 and Py3.
+        #
+        # https://docs.python.org/3/library/urllib.error.html
+        # https://docs.python.org/2/library/urllib.html
+        #
+        logger.error('caught non-fatal exception, see trace below')
+        logger.exception(err)
```

Comment:
```suggestion
        logger.exception('caught non-fatal exception while trying to update gensim-data cache from %r; using local cache at %r instead', DATA_LIST_URL, cache_path)
```

(no need for some many logging events, and stack trace already includes the original error).

Reply:
@menshikh-iv 's suggestion in https://github.com/RaRe-Technologies/gensim-data/issues/23 was to fail the command unless the user explicitly specifies an override. As opposed to silently falling back to other options (local cache).

Reply:
I don't think there is a significant difference between the two scenarios (override vs implicit cache), because the underlying data changes so infrequently.

Reply:
> I don't think there is a significant difference between the two scenarios (override vs implicit cache), because the underlying data changes so infrequently.

My proposal just least surprises (if you want to do - please do that explicitly), nothing more.

> Also, we're already using the local cache implicitly for the actual data, so why would we insist on using the cache explicitly (via an override) for the metadata?

let's discuss that in chat (I don't really get what you mean).